scripts/Lykke.Service.Tier.LimitUpdater/Program.cs

```diff
@@ -121,17 +122,26 @@ private static async Task Execute(string[] args)
                     continue;
                 }
 
+                var countryFromId = personalData.CountryFromID.Length == 2
```

Comment:
If properties CountryFromID or CountryFromPOA could be null (I don't know if it's possible with the current settings structure), it would be necessary to check for null or use safe navigation operator.